README.md

```diff
@@ -218,26 +223,39 @@ def authorization_callback():
     scopes = auth_token.scope
 
     # iterate scope entries here or check if a permission is in the scope
-    if "patient/Patient.read" in scopes: 
-        # patient info access granted
+    if "patient/Patient.r" in scopes: 
```

Comment:
It's a little arbitrary what this is really meant to represent, but I think this should really be
```suggestion
    if "patient/Patient.r" in scopes or "patient/Patient.rs" in scopes: 
```